### PR TITLE
Adding website guidelines checklist

### LINKED ISSUE #23: Add release checklist and versioning best practices to `howto`
Add this release checklist https://github.com/cncf/hugo-netlify-starter/pull/31 and the documentation on versioning best practices to https://github.com/cncf/techdocs/tree/main/howto

Context: https://github.com/cncf/techdocs/pull/19#issuecomment-867240741

### PR DESCRIPTION
**Pull request by nate-double-u**
Creating a website guidelines checklist based on https://github.com/cncf/foundation/blob/master/website-guidelines.md & https://github.com/cncf/hugo-netlify-starter/pull/31

Deploy preview: https://github.com/cncf/techdocs/blob/23-release-checklist/howto/website-guidelines-checklist.md

With https://github.com/cncf/techdocs/pull/25 merged, this closes #23

### CODE REVIEW

File: howto/website-guidelines-checklist.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] 4. Links to companies offering support go to a page that at least mentions
+  support of the project
+- [ ] 5. Copyright notice present at bottom of page.
+  Copyright to the project authors or to CNCF, not the origin company.
+- [ ] 6. CNCF Branding elements
+  - [ ] “We are a Cloud Native Computing Foundation project.” or “We are a Cloud
+  Native Computing Foundation sandbox project.” present (depending on status)  
+  - [ ] CNCF logo near the bottom of their project homepage  
+  - [ ] (optional) link to KubeCon + CloudNativeCon as the events approach  
+- [ ] 7. Footer contains:
```
Comment (amye) [inline howto/website-guidelines-checklist.md:28]: I need to review this for accuracy, there's something that tells me this is maybe old information. 
I cannot put my finger on it.

File: howto/website-guidelines-checklist.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "Copyright The Linux Foundation. All rights reserved. The Linux Foundation has
```
Comment (amye) [inline howto/website-guidelines-checklist.md:29]: Oh I know what it is! Copyright to authors, not LF!

File: howto/website-guidelines-checklist.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  [Trademark Usage](https://www.linuxfoundation.org/trademark-usage/) page."
+
+
+## Contributing guidelines and conduct pages
+
+The following files should be in the root of the repository:
+
+- [ ] Code of Conduct
+- [ ] Guidelines for Contributors (CONTRIBUTING.md or similar)
+- [ ] License file(s)
```
Comment (amye) [inline howto/website-guidelines-checklist.md:41]: Governance?

**Review by amye — COMMENTED** [review]
I think I see one area of improvement, and one nice-to-have.

File: howto/website-guidelines-checklist.md (hunk 1 above)
Comment (nate-double-u) [reply howto/website-guidelines-checklist.md:28]: Thanks for  looking at it -- I do have to do another pass doing a close read of the foundation's guidelines

File: howto/website-guidelines-checklist.md (hunk 2 above)
Comment (nate-double-u) [reply howto/website-guidelines-checklist.md:29]: Right, I'll get that changed out!

File: howto/website-guidelines-checklist.md (hunk 3 above)
Comment (nate-double-u) [reply howto/website-guidelines-checklist.md:41]: I'll add that too.

Comment (nate-double-u) [reply howto/website-guidelines-checklist.md:41]: Actually, do we need a GOVERNANCE file for websites, or just the main project repo? I'd think that the governance would be in the main repo, and would be written as a page in the website.

Comment (amye) [reply howto/website-guidelines-checklist.md:41]: Honestly, it's a nice to have! It's the main project repo usually as a markdown file so forget I said anything.

Comment (nate-double-u) [reply howto/website-guidelines-checklist.md:42]: I'dd add it in then, thanks!

File: howto/website-guidelines-checklist.md (hunk 1 above)
Comment (celestehorgan) [reply howto/website-guidelines-checklist.md]: @amye - I'm going to merge this (for now) and we can update later.

**Pull request merged**